src/Conversions.cc

```diff
@@ -568,6 +568,14 @@ msgs::Light ignition::gazebo::convert(const sdf::Light &_in)
   out.set_spot_inner_angle(_in.SpotInnerAngle().Radian());
   out.set_spot_outer_angle(_in.SpotOuterAngle().Radian());
   out.set_spot_falloff(_in.SpotFalloff());
+
+  // todo(anyone) Use the field isLightOn in light.proto from
```

Comment:
We're trying to reduce the `anyone`s in the code, can you add `ahcorde`? It would be good to make sure this gets addressed soon after the PR is ported to `main`.

Also

```suggestion
  // todo(anyone) Use the field is_light_off in light.proto from
```